lib/posix-process/signal/sigtimedwait.c

```diff
@@ -0,0 +1,57 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <signal.h>
+#include <stdlib.h>
+#include <sys/time.h>
+
+#include <uk/essentials.h>
+#include <uk/syscall.h>
+#include <uk/timeutil.h>
+
+#include "process.h"
+#include "signal.h"
+
+UK_SYSCALL_R_DEFINE(int, rt_sigtimedwait,
+		    const sigset_t *__unused, set,
```

Comment:
These `__unused` may be unnecessary as `UK_SYSCALL_R_DEFINE` already adds them.

Reply:
Yes, these are remains since it was a stub in `libuksignal`, will remove.